docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md

```diff
@@ -45,6 +45,23 @@ label_replace(
 
 For more information, refer to [Querying Prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/) in the Prometheus documentation.
 
+## Metrics collection level
+
+The metrics collection level setting controls how many database metrics are collected and stored. It helps balance system performance with monitoring needs, so that you can optimize resource usage based on your specific observability requirements.
+
+Set the metrics collection level by using the **Metrics Collection Level** Universe Runtime Configuration option (config key `yb.metrics.collection_level`). Refer to [Manage runtime configuration settings](../../../yugabyte-platform/administer-yugabyte-platform/manage-runtime-config/).
+
+You can set the collection level to the following:
+
+| Level | Description |
+|:-------|:---------|
+| ALL | Collects all metrics from the database. Not recommended for large databases with a lot of tables, as it can overload the YugabyteDB Anywhere (YBA) Prometheus instance and cause Out of Memory (OOM) errors on the YBA node.  |
```

Comment:
```suggestion
| ALL | Collects all metrics from the database. Not recommended for large databases with a lot of tables, as it can overload the YugabyteDB Anywhere Prometheus instance and cause Out of Memory (OOM) errors on the node where YugabyteDB Anywhere is installed.  |
```